### ISSUE TITLE
Check website resources and move any external to ORG server

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/cmp-issues#94]
Aids user privacy if these are stored on our server. Some of this got overlooked by me at launch.

**Comment by graphiclunarkid** [comment]
I believe @webal had to use the Bootstrap CMS rather than the local version because it broke locally for some reason. So that will need correcting at some point. Is there a tool we can use to spider the site and check for others?

**Comment by webal** [comment]
Yeah, can't remember exactly why but switching to the cdn got the menus working again in the mobile version. No reason why it wouldn't work if you copy down the same versions linked to on the cdn.

Not sure of a tool, but the only places I can think of are CSS & JS (header and footer templates) and possibly the sharing stuff, although I think you did that without any external links.
